recipes/biopandas/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "biopandas" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "976860772b6352ce13e5f31b4a43cd6ac6a4f3ac81f8a61867d9d7839ccd76e1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pandas
```

Comment:
Usually, when using `--single-version-externally-managed`, we don't need the run time dependencies to be available at build time (`numpy` and `pandas`) in this case. Can you try removing them? If it does not work just add them back, sometime the authors import them in `setup.py` and they must be present.